.github/workflows/node.js.yml

```diff
@@ -47,3 +45,21 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - run: npx @adobe/sizewatcher
+  semantic-release:
+    runs-on: ubuntu-latest
+    needs: [build]
+    if: ${{  !contains(github.event.head_commit.message, '[ci skip]') && github.ref == 'refs/heads/master' }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+      - name: Use Node.js 12.19
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.19'
+      - run: npm install
+      - run: npm run build
+      - run: npm run semantic-release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
we use this github token: https://github.com/adobe/asset-compute-pipeline/blob/master/.github/workflows/node.js.yml#L62-L63
```
GITHUB_TOKEN: ${{ secrets.ADOBE_BOT_GITHUB_TOKEN }}
```

Reply:
@jdelbick I'm not sure what the difference is. The "old" release process that I mentioned used `GITHUB_TOKEN`, so that's why I used it here as well.